package.json

```diff
@@ -33,6 +33,8 @@
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && ./scripts/publish-docs",
     "flow": "eslint packages",
+    "todo": "leasot 'packages/*/src/**.js'",
```

Comment:
wouldn't this be `todo:build` or `build:todo`?

otherwise the command is strange